spells.yml

```diff
@@ -8695,7 +8695,7 @@ storm_of_vengeance:
 
     **Torrential Rain.** A deluge of water pours from the sky. Any open fires in the area are extinguished, the ground becomes difficult terrain for 1 minute, and any holes less than 10 feet deep are filled with water. The area is heavily obscured by the driving rain until the start of your next turn.
 
-    **Crashing Lightning.** Tremendous bolts of lightning strike up to six points beneath the cloud of your choice that you can see. Each creature or object within 5 feet of one or more points must succeed on a Dexterity saving throw or take 10d6 lightning damage, or half as much on a success. Additionally, each creature within 100 feet of one or more points must succeed on a Constitution saving throw or take 2d6 thunder damage and become deafened for 5 minutes.
+    **Crashing Lightning.** Tremendous bolts of lightning strike up to six points beneath the cloud of your choice that you can see. Each creature or object within 5 feet of one or more points must succeed on a Dexterity saving throw or take 10d6 lightning damage, or half as much damage on a successful one. Additionally, each creature within 100 feet of one or more points must succeed on a Constitution saving throw or take 2d6 thunder damage and become deafened for 5 minutes.
```

Comment:
Actually this phrasing is better with "on a success" imo (but ty for fixing the "damage").
I like this form because it's concise, but if you want you could make it the "make a save. On fail, X, else Y." format

Reply:
Why should it be inconsistent with every other spell?

Reply:
The first part of the sentence is different: "[a target] must succeed on [a saving throw] or take [damage], or half as much damage on a success" vs. "[a target] must make [a saving throw]. On a failed save, a target takes [damage], or half as much damage on a successful one"

Reply:
I don't think that existing language is particularly good, honestly. "You must succeed or x, or y on a success" really does not make sense. I'm uploading a new commit that just rewords that whole bit.